modules/ROOT/pages/tutorial/access-control.adoc

```diff
@@ -83,10 +82,10 @@ This same database would be used by a number of different users, each with diffe
 
 Unlike applications which often require users to be modeled within the application itself, databases provide user management resources such as roles and privileges.
 This allows users to be created entirely within the database security model, a strategy that allows the separation of access to the data and the data itself.
-For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/[Cypher Manual -> Access control].
+For more information, see xref:authentication-authorization/index.adoc[].
 
 The following examples show two different approaches to using Neo4j security features to support the _healthcare_ database application.
-The first approach uses xref:authentication-authorization/built-in-roles.adoc#auth-built-in-roles[Built-in roles], whereas the second uses more advanced resources with fine-grained privileges for <<auth-access-control-using-privileges, sub-graph access control>>.
+The first approach uses xref:authentication-authorization/built-in-roles.adoc#auth-built-in-roles[], whereas the second uses more advanced resources with fine-grained privileges for <<auth-access-control-using-privileges, sub-graph access control>>.
```

Comment:
The explicit removal of `Built-in roles` in these `[]` definitely makes me think the other places are fine as well and that some of the comments about removing `Cypher Manual ->` could also just have empty brackets 🤷

Reply:
Actually, [] doesn't work when it points to an anchor, so I am adding the Heading back.